packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -358,47 +360,296 @@ class AppKitView extends _DarwinView {
+///
+/// ## Pointer events
+///
+/// `HtmlElementView` will interfere with Flutter's normal gesture detection. In
+/// order for the `HtmlElementView` contents to be interactive, they're allowed
+/// to handle `pointer-events`. This may result in Flutter missing some events
+/// because they've been handled by the `HtmlElementView`, and never seen by
+/// Flutter.
+///
+/// [`package:pointer_interceptor`](https://pub.dev/packages/pointer_interceptor)
+/// may help in some cases where Flutter content needs to be overlaid on top of
+/// an `HtmlElementView`. Alternatively, the `pointer-events: none` property can
+/// be set `onElementCreated`; but that will prevent **ALL** interactions with
+/// the underlying HTML content.
+///
+/// If the `HtmlElementView` is an `<iframe>` element, Flutter will not be able
+/// to see any pointer events that land in the `<iframe>` (click/tap, drag,
+/// drop...). In those cases, the HTML View will seem like it's "swallowing"
+/// the events and not participating in Flutter's gesture detection.
+///
+/// `package:webview_flutter` for the web is implemented as an `HtmlElementView`
+/// backed by an `<iframe>` element.
+///
+/// ## Visibility
+///
+/// `HtmlElementView` objects have an `isVisible` property that can be passed when
+/// registering the factory function, or calling `fromTagName`. In this case,
+/// the "visibility" refers to whether the `HtmlElementView` will paint pixels
+/// or not.
+///
+/// Correctly defining this value helps the Flutter web rendering engine optimize
+/// the amount of "overlays" it'll need to render a particular scene; some browsers
+/// can only have a limited number of overlays on a single page at a given time.
+///
+/// In general, `isVisible` should be left to its default value of `true`, but
+/// in some `HtmlElementView`s (like the `pointer_interceptor` or `Link` widget),
+/// it can be set to `false`, so the engine doesn't "waste" an overlay to render
+/// Flutter content on top of views that don't paint any pixels.
+///
+/// ## Accessibility
+///
+/// Flutter web enables accessibility by waiting for a double-tap in a
+/// button that covers the full page.
+///
+/// Placing this button in front of the scene would cause `HtmlElementView`s to
+/// not receive any pointer events, so the button is sent to the back.
+/// `HtmlElementView`s interferes with normal pointer events (See **Pointer
+/// events** above), so in order to enable accessibility, a **user must be able
+/// to double tap the app outside of the root `element` of an `HtmlElementView`**.
+///
 /// Make sure that your HTML views are sized no larger than necessary, or you
 /// may cause difficulty for users trying to enable accessibility.
 ///
-/// {@macro flutter.widgets.AndroidView.lifetime}
+/// Alternatively, a Flutter web app may programmatically enable accessibility
+/// by calling [SemanticsBinding.ensureSemantics].
```

Comment:
Not sure if this is the right place to document this 🤔

Reply:
Probably there's a better place, and then we should forward to the better place from here. The thing is that currently this is documented NOWHERE:

https://www.google.com/search?q=%22ensureSemantics%22+site%3Aflutter.dev